### DISCUSSION TITLE
Unlinked mentions

### DISCUSSION THREAD

**Discussion by HusseinT2000** [notaapp/talk#163] [Feature Requests] [answered]
I'm really enjoying Nota and since switching between different note apps, Nota feels the most comfortable use

I have found Obsidian's unlinked mentions extremely helpful for when I decide to create a new page after having referred to it previously.

Currently I make pages in advance in case I may require them later.

For example, currently at uni we are doing upper limb anatomy and early on a muscle was mentioned a few times but one of our lectures had a heavy focus on that muscle and it would be really beneficial if at the bottom there was a list of pages where it was mentioned and I could click a button to easily turn that mention into a link to the current page.

Thank you!

**Comment by astoilkov** [comment]
Yes. I agree.

We are making progress toward that feature. It will probably land in one of the next few releases.

Thanks for writing to us!

**Comment by webwesen** [comment]
agreed, unlinked references are very powerful.

**Comment by astoilkov** [comment]
It's probably a good place to mention that we are also experimenting with "related notes" feature. The idea is to show you notes that are related to the currently opened note. The algorithm extracts keywords and topics from each note and shows you relevant notes.

**Reply by astoilkov** [reply]
Screenshot of the private experiment:
![CleanShot 2022-05-07 at 09 23 43@2x](https://user-images.githubusercontent.com/884810/167241728-383fdeec-12fe-43cf-8053-87def6d95641.png)

Note that none of these notes have backlinks that connect them. I also have many other notes that include "react" in the title or in the text (e.g. `how should I react when...`).

**Reply by HusseinT2000** [reply]
Honestly I think this might be better actually, nice work!

**Reply by webwesen** [reply]
would be also great if the right hand side bar could show a bit more context, if, say, 'react' is in the h2 header - it might show a few lines of that section (or entire section between h2's

my use case - meeting notes in journal pages with more contextual content 'blocks' in peoples' pages
thanks for the great tool - paid already :)

**Reply by astoilkov** [reply]
Yes. I agree. Text from the file makes a lot of sense. Sometimes the file name doesn't match the contents very well and you are left confused of why this note shows up.

**Comment by astoilkov** [comment, marked as answer]
Unlinked mentions are now part of the latest release. You can find them in the Go to Mention... dialog (⌘⇧B). The next release will probably include a right sidebar experiment where you will be able to see both backlinks and unlinked mentions.

**Reply by NoodleNodes** [reply]
Yes please 😊🙏

**Comment by NoodleNodes** [comment]
> It's probably a good place to mention that we are also experimenting with "related notes" feature. The idea is to show you notes that are related to the currently opened note. The algorithm extracts keywords and topics from each note and shows you relevant notes.

Take a look at [napkin](https://www.napkin.one) which uses AI to achieve something similar to what you are proposing.

**Comment by astoilkov** [comment]
The next release is out and unlinked mentions can also be seen in the right sidebar!
